lib/Bio/EnsEMBL/DataCheck/Checks/TimelySemaphoreRelease.pm

```diff
@@ -0,0 +1,85 @@
+=head1 LICENSE
+
+Copyright [2018-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::TimelySemaphoreRelease;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Data::Dumper;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'TimelySemaphoreRelease',
+  DESCRIPTION    => 'Check that every semaphored job was started only after all of its fan jobs had completed.',
+  GROUPS         => ['compara_gene_tree_pipelines'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['compara'],
+  TABLES         => ['job', 'semaphore']
+};
+
+sub tests {
+    my ($self) = @_;
+    # A typical test may use the sql_helper:
```

Comment:
This seems like it could be deleted without harm:
```suggestion
```

Reply:
I will delete this line.